apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -337,7 +345,7 @@ export class FilesStorageUC {
 			});
 
 			await Promise.all(pendedFileRecords);
-			return [newRecords, newRecords.length];
+			return responseEntities;
```

Comment:
On this place i am not sure. Copy is a operation for one domain object yes. But for the file services this object do not exist. For the file service it is a scope that it should search and i can find nothing, one or many element for this. Also a find operation that should result in a paginated (Counted) result from what is expected on api side.